libraries/AP_HAL_SITL/RCOutput.cpp

```diff
@@ -86,4 +96,31 @@ void RCOutput::push(void)
     }
 }
 
+/*
+  setup output mode
+ */
+void RCOutput::set_output_mode(enum output_mode mode)
+{
+    if (_output_mode == mode) {
```

Comment:
This is only here to facilitate the debug?  Rearrange to remove this if?

Reply:
and copied from pixhawk hal